ratelimit/http_test.go

```diff
@@ -107,14 +110,20 @@ func TestWithLocalRateLimiter(t *testing.T) {
 
 	for _, c := range cases {
 		t.Run(c.name, func(t *testing.T) {
-			rlmw := WithLocalRateLimiter(c.configs...)
```

Comment:
I have modified the test file to replicate the order in which the middlewares are executed in metrics/v1/http.go